slkarsh/color-picker-be
# Setup server

#### What's this PR do?

Sets up the app.js and server.js files, creates a migration for the projects and palettes tables, and seeds sample data for a project and its palettes.

#### Where should the reviewer start?

app.js file (which is then imported into server.js)

#### How should this be manually tested?

Make sure to install the dependencies added (CORS and babel have been added) and `npm start` to ensure the server is running properly and the dependencies were installed (will receive an error if you're missing babel as the `import` syntax is used). Also run `knex migrate:latest` and `knex seed:run` to ensure tables are set up properly. These can be visualized in the terminal or Postico.
